src/hotspot/share/runtime/safepoint.cpp

```diff
@@ -95,7 +95,7 @@ static void post_safepoint_synchronize_event(EventSafepointStateSynchronization&
                                              uint64_t safepoint_id,
                                              int initial_number_of_threads,
                                              int threads_waiting_to_block,
-                                             uint64_t iterations) {
+                                             int iterations) {
```

Comment:
Maybe leave iterations as unsigned and change the caller instead?

Reply:
If I change the caller: synchronize_threads, it returns the parameter iterations, so I'd have to change its callers.  Everything uses iterations here as an int, other than this parameter.